internal/fs/handle/file.go

```diff
@@ -165,6 +165,17 @@ func (fh *FileHandle) Read(ctx context.Context, dst []byte, offset int64, sequen
 	return
 }
 
+// To propagate the mode in which file has been opened while calling the Write()
```

Comment:
the comment is not clear. what are you trying to explain.
Why we are adding Write() method in fileHandle.

Mention that all writes will be done at indoe level. Method is added in fileHandle to be able to pass the fileOpenMode which is used for determining the write path.
For eg: in case of append mode for ZB unfinalized objects, streaming writes flow is used.